platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -438,19 +385,32 @@ public void initialize(
     roleFilter = (String) options.get(ROLE_FILTER);
     roleNameAttribute = (String) options.get(ROLE_NAME_ATTRIBUTE);
     roleSearchSubtree = Boolean.parseBoolean((String) options.get(ROLE_SEARCH_SUBTREE));
-    startTls = Boolean.parseBoolean(String.valueOf(options.get(SSL_STARTTLS)));
     setBindMethod((String) options.get(BIND_METHOD));
     realm = (String) options.get(REALM);
     kdcAddress = (String) options.get(KDC_ADDRESS);
+    String connectionPoolId = (String) options.get(CONNECTION_POOL_ID);
+    installLdapConnectionPool(connectionPoolId);
```

Comment:
We could simplify the SslLdapLoginModule class and not have to pass as many properties around by moving all the logic to LdapLoginConfig and just pass a service reference to this class and call the corresponding methods required. May save that for another ticket though if people like the idea...